test/boilerplate_suggestors/props_meta_migrator_test.dart

```diff
@@ -58,17 +58,30 @@ main() {
       test('', () {
         converter.setConvertedClassNames({
           'FooProps': 'FooProps',
+          'BarProps': 'BarPropsMixin',
         });
 
         testSuggestor(
-          expectedPatchCount: 2,
+          expectedPatchCount: 7,
           input: '''
+          /// Some doc comment
+          @PropsMixin()
+          abstract class FooPropsMixin implements UiProps {
+            // To ensure the codemod regression checking works properly, please keep this
+            // field at the top of the class!
+            // ignore: undefined_identifier, undefined_class, const_initialized_with_non_constant_value
+            static const PropsMeta meta = _\$metaForFooPropsMixin;
+            
+            String foo;
+          }
+              
           @Component2()
           class FooComponent extends UiComponent2<FooProps> {
+            static const foo = FooProps.meta;
```

Comment:
Curious: does this really happen often enough to justify a codemod?

Reply:
🤷‍♂ its done though... so... does it matter?

Reply:
Sorry, I meant that purely out of curiosity. If it did matter I'd be more direct 😜